mate-panel/panel-toplevel.c

```diff
@@ -4220,6 +4220,10 @@ panel_toplevel_finalize (GObject *object)
 static void
 panel_toplevel_class_init (PanelToplevelClass *klass)
 {
+	GdkDisplay *display;
+
+	display = gdk_screen_get_display(gdk_screen_get_default());
```

Comment:
formating...
```suggestion
	display = gdk_screen_get_display (gdk_screen_get_default());
```

Reply:
Format fixed in case we want to keep working on this